src/scales/scale.radialLinear.js

```diff
@@ -380,14 +380,11 @@ class RadialLinearScale extends LinearScaleBase {
 
 	getIndexAngle(index) {
 		var chart = this.chart;
-		var angleMultiplier = 360 / chart.data.labels.length;
+		var angleMultiplier = Math.PI * 2 / chart.data.labels.length;
 		var options = chart.options || {};
 		var startAngle = options.startAngle || 0;
 
-		// Start from the top instead of right, so remove a quarter of the circle
-		var angle = (index * angleMultiplier + startAngle) % 360;
-
-		return (angle < 0 ? angle + 360 : angle) * Math.PI * 2 / 360;
+		return _normalizeAngle(index * angleMultiplier + toRadians(startAngle));
```

Comment:
It's a bit surprising to me that this is returning radians now that all the options are specified in degrees. Should we just standardize on degrees everywhere?

Reply:
`CanvasRenderingContext2d` is going to need radians as input, so need to convert at some point. 
IMO its clear that options are degrees, while everything internal (API included) should be radians.

There are only couple of things that contribute to that opinion:
 * developers are usually familiar with radians
 * people in general (users) are more likely familiar with degrees.
 * most functions require radians as input

Reply:
Ok. That makes sense. Should we convert `determineLimits` to use radians and remove the `toDegrees` function in that case?